core/ecschema-editing/src/test/Editing/Mixins.test.ts

```diff
@@ -26,4 +26,29 @@ describe("Mixins tests", () => {
     const mixinResult = await testEditor.mixins.create(testKey, "testMixin", entityKey);
     expect(testEditor.schemaContext.getSchemaItemSync(mixinResult.itemKey!)?.name).to.eql("testMixin");
   });
+
+  it("should delete a mixin", async () => {
+    const schema = await testEditor.schemaContext.getCachedSchema(testKey);
+    const className = "testMixin";
+    const mixinResult = await testEditor.mixins.create(testKey, className, entityKey);
+    const mixin = await schema?.getItem(className);
+    expect(mixin).not.undefined;
+
+    /* eslint-disable @typescript-eslint/no-non-null-asserted-optional-chain */
```

Comment:
why is this needed?  Why couldn't we right the code so it does not violate this lint rule?  Same for the the next case where this rule is ignored.

Reply:
Done